backend/core/src/listings/listings.service.ts

```diff
@@ -23,15 +23,24 @@ export class ListingsService {
   constructor(@InjectRepository(Listing) private readonly listingRepository: Repository<Listing>) {}
 
   private getQueryBuilder() {
-    return this.listingRepository
-      .createQueryBuilder("listings")
+    return Listing.createQueryBuilder("listings")
```

Comment:
I don't understand this (maybe @avaleske does?) but switching `Listing.createQueryBuilder` back to `this.listingRepository.createQueryBuilder` fixes the issue with listings.service.spec.ts:

"ConnectionNotFoundError: Connection "default" was not found."

It's something about how the different repositories establish connections (through TypeORM) to the database, and maybe we were getting the test error because we were using a default behavior? There may be a better way to fix this, but that's what I've got for now 🙃

Reply:
Yeah that's the fix.

`Listing.createQueryBuilder...` uses the listing entity directly, but `this.listingRepository...` uses the injected entity, which lets the tests mock out the db.

Reply:
(see the `listingRepository: Repository<Listing>` in the constructor params on line 23 for the injected entity)

Reply:
Amazing, thanks!!